Sources/SKLogging/SplitLogMessage.swift

```diff
@@ -10,14 +10,16 @@
 //
 //===----------------------------------------------------------------------===//
 
+private let maxChunkSize = 800
```

Comment:
Doesn't seem this is used outside of `splitLongMultilineMessage`, was factoring out a leftover change from a previous revision?

Reply:
Yes, I think so. Moved it to `splitLongMultilineMessage`.